config/peerpods/peerpodssecret.yaml

```diff
@@ -10,4 +10,4 @@ stringData:
   #LIBVIRT_NET: "default
   #LIBVIRT_POOL: "default"
   #REDHAT_OFFLINE_TOKEN: "" #Required to download rhel base image : Download token from https://access.redhat.com/management/api
-  
\ No newline at end of file
+  #HOST_KEY_CERTS: "" #Download the certificate from https://www.ibm.com/support/resourcelink/api/content/public/host-key-documents.html
```

Comment:
@ANJANA-A-R-K  as I understand this is the cert content (as string) pasted in the secret. Have you tested it once to ensure that multi-line string is handled correctly ?

Reply:
@bpradipt  This has been tested by adding HOSTKEY in peer-pod-secret.yaml file.

@ANJANA-ARK Can you add all logs here.

Reply:
Yes we have tested it, we're providing the crt value like
```
stringData:
  CLOUD_PROVIDER: ""
  LIBVIRT_URI: ""
  LIBVIRT_POOL: ""
  LIBVIRT_VOL_NAME: ""
  REDHAT_OFFLINE_TOKEN: ""
  HKD_CRT: |
    -----BEGIN CERTIFICATE-----
    .
    .
    .
    -----END CERTIFICATE-----
```
this is the packer log where HKD is picked up the cloud api adaptor
```
==> qemu.rhel: Uploading qcow2/build-s390x-se-image.sh => ~/build-s390x-se-image.sh
==> qemu.rhel: Provisioning with shell script: /tmp/packer-shell3632283365
    qemu.rhel: Building SE podvm image for s390x
    qemu.rhel: Finding host key files
    qemu.rhel: found host key file: "/tmp/files/HKD.crt"

```
and we've tested by deploying a workload 
```
[root@bastion-sl ~]# oc get pods -n default
NAME             READY   STATUS    RESTARTS   AGE
osc-caa-commit   1/1     Running   0          100s

```

Reply:
AFAIU user needs to make sure the certificate lines are aligned, otherwise, it may mess it, maybe worth to add a comment..